### ISSUE TITLE
Do not show "no description found"

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#1054]
Even with #1034 fixed there will always be a lot of items with no description.

When no description is found, I think we should just leave the subtitle empty, rather than writing "no description found" to it. I believe "no description found" clutters the screen with a lot of non-information in both map and list modes. The label is always here anyway, so it is not like the item would lose its visual presence.

What do you think?

![screenshot_20180111-115556](https://user-images.githubusercontent.com/99590/34806495-aa219246-f6c6-11e7-8d49-15b180989ef0.png)

**Comment by misaochan** [comment]
Personally I think that's a great idea :)

**Comment by mithlesh4257** [comment]
Can I work on this issue?

**Comment by mithlesh4257** [comment]
Can we inplace of "no description" paste one of the search result of **gigablast**?

**Comment by maskaravivek** [comment]
@mithlesh4257 feel free to take up this issue. I guess removing the the "no description" text would be sufficient. Adding any other search result would again clutter the screen.

**Comment by misaochan** [comment]
I agree with @maskaravivek - search engine results could be an interesting potential enhancement to debate (with its own pros and cons), but removing the text should suffice for the time being. We can create another issue to debate the search engine query. :)

Please feel free to work on this issue @mithlesh4257 , thanks!

**Comment by mithlesh4257** [comment]
@nicolas-raoul  Sir Instead of messing with "no description" codes like XML or Java code, should I only set visibility false in Login activity xml, or what to do Sir please help Sir.

**Comment by tanvidadu** [comment]
@mithlesh4257 u still working on it. @misaochan if its up for grabs I will like to work on it.

**Comment by misaochan** [comment]
Please feel free to go for it, @tanvidadu , as we have not heard back from @mithlesh4257 in a while.

**Comment by tanvidadu** [comment]
I have fixed the error @misaochan but I made a noob error when I reforked my repo. I forgot to change my branch while fixing #1251 as a result when I make a new branch other commits are also visible. Once my pr for the above issue is merged i will add pr for this one.
Sorry for delay !
Mocks 
![nodescription](https://user-images.githubusercontent.com/29161745/37219165-9f7c873e-23e8-11e8-86f6-1df7ad3ba1a9.jpeg)
